Resources/Prototypes/_Impstation/Entities/Mobs/NPCs/miscellaneous.yml

```diff
@@ -168,3 +168,44 @@
     interactSuccessSound:
       path: /Audio/_Impstation/Animals/mwah.ogg
     interactSuccessSpawn: EffectHearts
+
+- type: entity
+  parent: PumpkinLantern
```

Comment:
this isn't a mob, can you put it in the appropriate directory? based on the parent, this would be `Resources/Prototypes/_Impstation/Entities/Objects/Decoration/jackolantern.yml`

Reply:
moved it!